### PR TITLE
Refactor build tasks

### PR DESCRIPTION
**Pull request by taka-sho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,19 +62,20 @@
   "scripts": {
     "build": "npm-run-all -p build:*",
     "build:gulp": "gulp build",
-    "build:webpack": "cross-env NODE_ENV=production webpack",
+    "before-gulp:pug": "pug app/*.pug --out dist/",
+    "before-gulp:webpack": "cross-env NODE_ENV=production webpack",
```
Comment (9renpoto) [inline package.json]: please refactor cmd name!

File: webpack.base.js [hunk 2]
```diff
@@ -9,6 +9,13 @@ exports.moduleConfig = {
       test: /\.vue$/,
       exclude: /node_modules/,
       loader: 'vue-loader'
+    },
+    {
+      test: /\.css$/,
+      loader: [
```
Comment (9renpoto) [inline webpack.base.js]: use ? https://webpack.js.org/guides/migrating/#chaining-loaders

Comment (taka-sho) [inline webpack.base.js]: to load [main.css](https://github.com/uncovertruth/userdive-tag-paster/blob/master/app/styles/main.css)

Comment (9renpoto) [inline webpack.base.js]: `s/loader/use/`

**Pull request merged**